.gitignore

```diff
@@ -9,7 +11,16 @@ Gemfile.lock
 .settings/
 .vscode/
 
-## Inproper copy when switching branch
+## IntelliJ
+
+.idea/
+
+## Improper copy when switching branch
+
+src/
+
+## Maven build files
 
 target/
-src/ 
\ No newline at end of file
+temp/
+dependency-reduced-pom.xml
```

Comment:
dependency-reduced-pom is gitignored on main branch yes, but i don't think it is removed completely. I would suggest you remove it from your fork, unless it is needed for something.